lib/evaluate.js

```diff
@@ -88,24 +90,24 @@ export const evaluate = async ({
     logger.log('EVALUATE ROUND %s: added %s as rounding to MAX_SCORE', roundIndex, delta)
   }
 
-  // Calculate aggregates per fraud detection outcome
+  // Calculate aggregates per evaluation outcome
   // This is used for logging and telemetry
-  /** @type {Partial<Record<import('./typings.js').FraudAssesment, number>>} */
-  const fraudAssessments = {
+  /** @type {Partial<Record<import('./typings.js').TaskingEvaluation | import('./typings.js').CommitteeCheckError, number>>} */
+  const evaluationOutcomes = {
     OK: 0,
     TASK_NOT_IN_ROUND: 0,
     DUP_INET_GROUP: 0,
     TOO_MANY_TASKS: 0
   }
   for (const m of measurements) {
-    fraudAssessments[m.fraudAssessment] = (fraudAssessments[m.fraudAssessment] ?? 0) + 1
+    evaluationOutcomes[m.taskingEvaluation] = (evaluationOutcomes[m.taskingEvaluation] ?? 0) + 1
   }
   logger.log(
-    'EVALUATE ROUND %s: Evaluated %s measurements, found %s honest entries.\n%o',
+    'EVALUATE ROUND %s: Evaluated %s measurements, accepted %s entries to reward.\n%o',
```

Comment:
```suggestion
    'EVALUATE ROUND %s: Evaluated %s measurements, rewarding %s entries.\n%o',
```

To avoid confusion with "accepted measurements"

Reply:
Or should it rather be just "accepted measurements"?

```suggestion
    'EVALUATE ROUND %s: Evaluated %s measurements, accepted %s entries.\n%o',
```

Reply:
I like _rewarding %s entries_ 👍🏻